app/src/main/res/layout/view_player_details.xml

```diff
@@ -17,17 +18,24 @@
 
     <View
         android:id="@+id/colorIndicatorWhite"
+        android:layout_width="6dp"
+        android:layout_height="match_parent"
+        android:background="@color/white"
+        app:layout_constraintLeft_toLeftOf="parent" />
+
+    <View
+        android:id="@+id/stoneColorSeparator"
         android:layout_width="1px"
         android:layout_height="match_parent"
         android:layout_marginLeft="6dp"
-        android:background="@color/black"
+        android:background="@color/buttonEnabled"
```

Comment:
in this particular case I would suggest we need to keep using a color black and color white, since this reffers to the color of the player. Let's say tomorrow we decide we want to have the buttons orange, we don't want the colour indicator to turn orange instead of black :)

Reply:
ah but this is not just replacing black
I used buttonEnabled because it is a c

Reply:
maybe we should name then "@color/whiteStone" and "@color/blackStone"

Reply:
could be confusing as the separator for white stones is actually black - I named it stoneColorSeparator for now
in any case this is one thingy that'll disappear once you migrate the game screen it won't last long